src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/Dependency.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+// protobuf vscode extension: https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3
+
+option java_multiple_files = true;
+option java_package = "com.microsoft.azure.functions.rpc.messages";
+option java_outer_classname = "DependencyProto";
+option csharp_namespace = "Microsoft.Azure.WebJobs.Script.Grpc.Messages";
+option go_package ="github.com/Azure/azure-functions-go-worker/internal/rpc";
+
+package AzureFunctionsRpcMessages;
+
+
+//Used to send the manaed dependency module for the language worker to download the client module
+message Dependency {
+  string name = 1;
+  string majorversion = 2; 
+  string path = 3;
+  bool isdownloadrequired = 4;
+}
+
+//Used to send the dependency management request to the language worker
+message DependencyManagementRequest {
```

Comment:
Can we have a clearer name for this message? What is the action expected from the worker when this message is received?

Reply:
As part of this message, the language worker downloads the mentioned dependencies. I can change the message name as "DependencyDownloadRequest"